app/src/main/java/com/opsat7373/moviestars/presentation/viewmodel/MoviesListViewModel.kt

```diff
@@ -0,0 +1,18 @@
+package com.opsat7373.moviestars.presentation.viewmodel
+
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import com.opsat7373.moviestars.data.model.Movie
+import com.opsat7373.moviestars.data.repository.MovieRepository
+
+class MoviesListViewModel : ViewModel() {
+    var moviesList = MutableLiveData<List<Movie?>>()
```

Comment:
not-mutable live data is preferred.
if you need Mutable, you can keep private field as mutable, but expose just LiveData via public field.
`
private val _moviesList = MutableLiveData<List<Movie>>()
`
`
val movieList: LiveData<List<Movie>> = _movieList
`